module4-authentication-and-security/r1.2-adding-authorization-layer/README.md

```diff
@@ -0,0 +1,222 @@
+# Adding authorization layer
+
+With authorization, we introduce another level of app security. In this lesson, we will learn how to restrict some users from accessing certain routes of our app. The lesson objectives are:
+
+- Define what is Authorization, and how is it different from Authentication
+- Introduce role-based authorization
+- How to restrict access to routes using authorization
+
+## **Authorization**
+
+- Authorization is a process by which a server determines if the client has permission to use a resource or access a file.
+- Authorization is usually coupled with authentication so that the server has some concept of who the client is that is requesting access.
+- The type of authentication required for authorization may vary; passwords may be required in some cases but not in others.
```

Comment:
not really an appropriate use of bullet points IMO...this should just be written as a paragraph.